src/sentry_plugins/github/endpoints/webhook.py

```diff
@@ -36,7 +36,7 @@ def __call__(self, organization, event):
         repo = Repository.objects.get_or_create(
             organization_id=organization.id,
             provider='github',
-            external_id=event['repository']['full_name'],
+            external_id=event['repository']['id'],
```

Comment:
Can we explicitly coerce to a text type here? This is an integer from the API but we're putting this into a charfield.